test/executable_test.dart

```diff
@@ -640,8 +640,9 @@ void main() {
             private: true
             environment:
               sdk: '>=2.4.0 <3.0.0'
+            dependencies:
+              logging: 1.0.0 <=2.0.0'
```

Comment:
Looks like this range isn't quite right. Let's just change this to:
```suggestion
              logging: 1.0.2
```